test/normalization/quotation/nested-no-siblings.js

```diff
@@ -0,0 +1,31 @@
+/** @jsx h */
+
+export const input = <editor>
```

Comment:
I think we should allow this:
```
<quote>
        <quote>
            <p><cursor/>Content</p>
        </quote>
    </quote>
```

Reply:
I don't understand what a quoted quote with no other context would mean, but it's something we can discuss. This is however consistent with our current behaviour.

Reply:
I imagine this like:
1. user create a quote
2. user q=create nested quote
3. user adds a paragraph after nested quote

Currently, he would have to add a paragraph and then wrap first para in a quote

Reply:
Sure, we could allow this, but what for? What would

> > such quote

mean in your opinion? How is it different from just a single quote?

Reply:
I mean, if we want to allow nested quotes then we should allow creating them without any special handling

Reply:
We could only run this normalization if the outer quote is not selected, like with empty inlines. If cursor is within the outer quote, we allow such nesting, otherwise we normalize like it is currently implemented.

Reply:
That seems reasonable